packages/notifi-react-card/lib/components/subscription/defaults.css

```diff
@@ -813,3 +807,7 @@
   text-align: right;
   color: #1448F3;
 }
+
+.NotifiIntercomChatMessage__avatar {
```

Comment:
can we also add a background-color in case the image is a broken link?

(would be good to see a screenshot of that too)

Reply:
hmm, tried to apply a background and it doesn't work, add onError in <img> to check if the src is invalid, it will show noting instead of a broken image.
<img width="446" alt="Screenshot 2022-12-01 at 10 40 23 PM" src="https://user-images.githubusercontent.com/26240001/205231660-2a269501-1ccc-4793-8abf-218e5ccfafaa.png">

Reply:
actually this feels better